packages/scripts/utils/config.js

```diff
@@ -354,6 +354,16 @@ function getWebpackEntryPoints( buildType ) {
  * @return {Array}  The list of all the `render` prop paths included in `block.json` files.
  */
 function getRenderPropPaths() {
+	return getPhpFilePaths( [ 'render' ] );
+}
```

Comment:
I assume we can't remove this function, right? Are these utils public? It seems `getRenderPropPaths` is not used anywhere internally.

Reply:
Good question. The function isn't exported from the `@wordpress/scripts` package, but it seems like we have some precedent of recommending importing functions from a subdirectory of that package, e.g. [here](https://github.com/WordPress/gutenberg/blob/8112ec2e82b6968f8699e672ae1a4b22897c58e8/packages/dependency-extraction-webpack-plugin/README.md#webpack):

> ```js
> const defaultConfig = require( '@wordpress/scripts/config/webpack.config' );
> ```

This is the reason why I didn't remove it, however slim chances are that someone actually does this with `getRenderPropPaths`.

However, your comment reminded me of the [`@wordpress/deprecated` package](https://github.com/WordPress/gutenberg/blob/8112ec2e82b6968f8699e672ae1a4b22897c58e8/packages/deprecated/README.md). Maybe I'll use that to annotate the function as deprecated, which should pave the way for removing it at a later time 🤔

Reply:
Okay 🙂 I agree probably nobody is using `getRenderPropPaths` but the "compatibility" code is not complex either.

Reply:
Ah, it's a bit of a chicken-egg situation 😬 Seems like we can't `require( '@wordpress/deprecated' )` in `@wordpress/scripts`, as the latter package is needed to build the former 😅

Reply:
I might just keep it as-is for now (to ensure I'm not breaking any obscure third-party thing that's relying on this).

Reply:
You should be able to remove `getRenderPropPaths`. We don't offer support for internal utils. I also don't expect this one would ever be necessary.

Reply:
Alright, I'll file a quick PR! Thank you for weighing in 😄